src/flows/templates/types/bucket.ts

```diff
@@ -12,18 +13,12 @@ export default register =>
         refresh: AUTOREFRESH_DEFAULT,
         pipes: [
           {
-            tags: {},
-            type: 'metricSelector',
-            field: '',
-            measurement: '',
-            aggregateFunction: {
-              name: 'mean',
-            },
-            bucket: {
-              name: name,
-            },
-            title: 'Select a Metric',
+            type: 'queryBuilder',
+            title: 'Build a Query',
             visible: true,
+            ...JSON.parse(
+              JSON.stringify(PIPE_DEFINITIONS['queryBuilder'].initial)
+            ),
```

Comment:
this seemed like the right thing to do here, but I am curious why the default query builder doesn't implement the defaults this way @drdelambre

Reply:
i'm 12% sure that you can pass in just `PIPE_DEFINITIONS['queryBuilder'].initial` because the code immediately after this deep clones as well, but not a big deal